### ISSUE TITLE
speed up training with multiple gpus

### ISSUE THREAD

**Issue by stephenplaza** [janelia-flyem/transfer_em#5]
Implement support for multiple GPU training using tensorflow's mirrored training.  This should be mostly straightforward except that the loss functions need to be modified to handle different actual batch size.
